### ISSUE TITLE
Anti-Doze: Advise users how to not `optimize` Pix-Art

### ISSUE THREAD

**Issue by Echolon** [kriztan/Pix-Art-Messenger#465]
Vendors are making the live hard for free software and developers in general. To counter their battery anti-features and increase the chance of making push possible again this thead should evaluate how to do it best.

In general the API from here may help us a lot to provide decent information for the user: https://dontkillmyapp.com/apidoc

Things which should be considered in my opinion:

- We can only expect to provide simple suggestion and steps to a user.
- The user must be aware that this is actually important. Also consider that most users were *never ever* aware that this is actually a problem.
- The user must be guided as much as possible (e.g. "simply" just open the settings menu page which is the correct one)
- Pix-Art is already showing and asking *a lot* during the onboarding which is basically good, but will also distract and likely annoy people and by this degrate the experience. Hence, before just implementing, it should be evaluated if the onboarding, introduction and optimization steps can be review and reduced in general. Maybe this screen should not just pop-up after installation but a few hours later? Maybe this screen should be pop-up several times? Maybe there should be a pinned hint that final "Optimization for Pix-Art" should be done? :smile:

**Comment by licaon-kter** [comment]
This is already implemented...users get prompted after install, right? Or at least most known broken OEMs

Foreground service is ON, right?

You want all the advice on that site implementet in dialogues per each OEM?

**Comment by Echolon** [comment]
Yes, I want to even more optimise that dialogues. However, doing that for every OEM is likely not feasible. But still I think there are points to improve in general.

**Comment by underhandedness** [comment]
Do not integrate into on boarding.  Impart this wisdom after the first message is sent.
